pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
+	pvcs := &core.PersistentVolumeClaimList{}
+	pvcLabels := map[string]string{
+		"migration": r.Migration.Name,
+		"ova":       "nfs-pvc",
+	}
+
+	err = r.Destination.Client.List(
+		context.TODO(),
+		pvcs,
+		&client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(pvcLabels),
+			Namespace:     r.Plan.Spec.TargetNamespace,
+		},
+	)
+	if err != nil {
+		return false, err
+	}
+
+	if len(pvcs.Items) == 0 {
+		return
+	}
+
+	pvc := &core.PersistentVolumeClaim{}
+	if len(pvcs.Items) > 1 {
+		for _, pvcVirtV2v := range pvcs.Items {
+			if pvcVirtV2v.CreationTimestamp.Time.After(r.Migration.CreationTimestamp.Time) {
```

Comment:
this code deserves a comment - is it because of leftovers we may have from previous migration attempts?

Reply:
yep exactly, it should't happen because we filter the PVC by migration name but still for any case, I"ll add comment.